File: 2022/programmazione/ppo/ppo.cpp
#include <stdio.h>
#include <iostream>
#include <string>

using namespace std;

int controlla(string nuova, string vecchia) {
    // SCRIVI QUA IL TUO CODICE
    // RITORNA 0 SE LA NUOVA PASSWORD NON SEGUE LE SPECIFICHE
    // RITORNA 1 SE LA NUOVA PASSWORD SEGUE LE SPECIFICHE 
    return 1;
}

int main() {

    // DECOMMENTA QUA SE VUOI LEGGERE DA FILE
    // freopen("input.txt", "r", stdin);   // File di input fornito dalla piattaforma
    // freopen("output.txt", "w", stdout); // File di output da caricare in piattaforma

    int N;
    string nuova, vecchia;

    cin >> N;

    for(int i=0; i<N; i++) {
        cin >> nuova >> vecchia;
        cout << controlla(nuova, vecchia) << endl;
    }

    return 0;
}

File: 2022/programmazione/ppo/ppo.py
#!/bin/env python3

import sys

# Se vuoi leggere/scrivere da file decommenta qua
# fin = open("input.txt", "r")  # File di input fornito dalla piattaforma
# fout = open("output.txt", "w")  # Output da inviare alla piattaforma

# Se vuoi leggere/scrivere da linea di comando decommenta qua
fin = sys.stdin  # input fornito dalla piattaforma
fout = sys.stdout  # Output da inviare alla piattaforma


def controlla(nuova, vecchia):
    print(nuova, vecchia)
    # SCRIVI QUA IL TUO CODICE
    # RITORNA 0 SE LA NUOVA PASSWORD NON SEGUE LE SPECIFICHE
    # RITORNA 1 SE LA NUOVA PASSWORD SEGUE LE SPECIFICHE
    return 1


N = int(fin.readline().strip())

for _ in range(N):
    nuova, vecchia = fin.readline().strip().split(" ")
    print(controlla(nuova, vecchia), file=fout)
